.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Find documentation here:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# @energietransitie/twomes-maintainers will be requested for
+# a review when someone opens a pull request.
+# change the line below to suit your own organization
+*   @energietransitie/twomes-maintainers
```

Comment:
@henriterhofte can you give twomes-maintainers access to this repo, to resolve this issue?

Reply:
I also did NOT push the updates to the workflows, since this means I don't have the rights to do that.

Reply:
I updated the access rights like you requested

Reply:
Thank you. That fixed the codeowners file and allowed my to push some commits.

Reply:
I added SECURE_BOOTLOADER_SIGNING_KEY as an environment secret to https://github.com/energietransitie/twomes-generic-esp-firmware/settings/secrets/actions... it seems not re-usable in other repos like this one. Did we decide to replicate the SECURE_BOOTLOADER_SIGNING_KEY in each repo for Twomes ESP32 firmware? I remember that organisational secrets were not usable for this...?